src/internal/nativeModule.web.ts

```diff
@@ -213,7 +213,7 @@ const getCurrentState = (
   } else if (type === NetInfoStateType.unknown) {
     const state: NetInfoUnknownState = {
       ...baseState,
-      isConnected: null,
+      isConnected,
```

Comment:
Okay, this looks probably good because it will reference something that is set here https://github.com/react-native-netinfo/react-native-netinfo/pull/544/files#diff-79b6fcc3eca6a8021e8e237993258e39190d1b01f917ee185caf17ba5c061b0dR106

and that should be safe because it is already in use and it appears to be supported since forever on the web https://www.w3schools.com/jsref/prop_nav_online.asp